static/js/containers/ProfileImage_test.js

```diff
@@ -149,7 +149,18 @@ describe('ProfileImage', () => {
         assert.isFalse(updateProfileImageStub.called);
       });
 
-      it('should disable the save button while uploading the image', () => {
+      it('should show a spinner while uploading the image', () => {
+        renderProfileImage(true, {
+          profile: thatProfile
+        });
+        helper.store.dispatch(startPhotoEdit({name: 'a name'}));
+        helper.store.dispatch(setPhotoDialogVisibility(true));
+        helper.store.dispatch(requestPatchUserPhoto(SETTINGS.user.username));
+        let dialog = document.querySelector(".photo-upload-dialog");
+        assert.isNotNull(dialog.querySelector('.mdl-spinner'));
```

Comment:
Is it worth asserting that the spinner isn't present during earlier stages of the editing/upload process?